docs/docs-app/constants/pages.js

```diff
@@ -30,8 +30,8 @@ export const examplePages = generatePath([
     children: [{
       name: 'Showcase',
       content: {
-        pageType: 'example',
         markdown: getDocUrl('examples/showcase.md'),
+        pageType: 'example',
```

Comment:
i started to make a more complex change but decided to move it to a different diff, this is an artifact of me not being able to remember the state before